private/serialization.nim

```diff
@@ -773,6 +806,27 @@ proc load*[K](input: Stream | string, target: var K) =
     elif e.parent of YamlParserError: raise (ref YamlParserError)(e.parent)
     else: internalError("Unexpected exception: " & e.parent.repr)
 
+proc loadMultiDoc*[K](input: Stream, target: var seq[K]) =
```

Comment:
please extend that to `input: Stream | string`, which is supported by the parser in devel.